### ISSUE TITLE
Show stable version by default

### ISSUE THREAD

**Issue by micmania1** [silverstripe/addons.silverstripe.org#103]
Related to: https://github.com/micmania1/silverstripe-blogger/issues/53#issuecomment-74441842

It shows the unstable versions first and doesn't differentiate from stable releases. Unstable releases should be in a different tab and a warning should be displayed when there are no stable releases.

![image](https://cloud.githubusercontent.com/assets/881537/6205416/449ab156-b5d2-11e4-97f5-d588684b50e4.png)

**Comment by camfindlay** [comment]
Yes, this sounds like a good feature to have. Almost need a design concept for the addons site.

**Comment by robbieaverill** [comment]
Good idea. It also would prevent us from encouraging users to install with the dev-master constraint.
